xclim/indices/generic.py

```diff
@@ -348,6 +376,21 @@ def get_dist(dist):
     return dc
 
 
+def get_lm3_dist(dist):
+    """Return a distribution object from `lmoments3.distr`."""
+    import lmoments3.distr
+
```

Comment:
```suggestion
```
Without this space, the comment about the manual installation will appear in conventional tracebacks, if the function is called without an installation of lmoments3.
We could also catch the error and raise an ImportError with the same message, but I don't think it is necessary here.